habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -186,20 +187,78 @@ def __init__(
         obs_transform=ResizeCenterCropper(size=(256, 256)),
     ):
         super().__init__()
-        self.goal_sensor_uuid = goal_sensor_uuid
 
         self.prev_action_embedding = nn.Embedding(action_space.n + 1, 32)
         self._n_prev_action = 32
+        rnn_input_size = self._n_prev_action
+
+        if (
+            IntegratedPointGoalGPSAndCompassSensor.cls_uuid
+            in observation_space.spaces
+        ):
+            n_input_goal = (
+                observation_space.spaces[
+                    IntegratedPointGoalGPSAndCompassSensor.cls_uuid
+                ].shape[0]
+                + 1
+            )
+            self.pointgoal_gps_compass_embedding = nn.Linear(n_input_goal, 32)
+            rnn_input_size += 32
 
-        self._n_input_goal = (
-            observation_space.spaces[self.goal_sensor_uuid].shape[0] + 1
-        )
-        self.tgt_embeding = nn.Linear(self._n_input_goal, 32)
```

Comment:
I can update the released model weights once this lands, but name changes like this will require that

Reply:
I will use old module name.